src/community/schemaless-features/mongodb-schemaless/src/test/java/org/geoserver/schemalessfeatures/mongodb/WFSSchemalessMongoTest.java

```diff
@@ -92,7 +93,7 @@ public void testGetStationFeaturesWithFilterPOST() throws Exception {
                 postAsJSON(
                         "wfs?request=GetFeature&version=1.1.0&typename=gs:"
                                 + StationsTestSetup.COLLECTION_NAME
-                                + "&outputFormat=application/json&cql_filter=measurements.values.value > 2000",
```

Comment:
Removed because the measurements is an array that can contain multiple values arrays, and GeoServer can not handle such situations (as I understand)